bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/ThingFactoryHelper.java

```diff
@@ -184,12 +185,17 @@ public static ChannelBuilder createChannelBuilder(ChannelUID channelUID, Channel
             label = channelType.getLabel();
         }
 
+        AutoUpdatePolicy autoUpdatePolicy = channelDefinition.getAutoUpdatePolicy();
+        if (autoUpdatePolicy == null) {
+            autoUpdatePolicy = channelType.getAutoUpdatePolicy();
+        }
+
         final ChannelBuilder channelBuilder = ChannelBuilder.create(channelUID, channelType.getItemType()) //
                 .withType(channelType.getUID()) //
                 .withDefaultTags(channelType.getTags()) //
                 .withKind(channelType.getKind()) //
                 .withLabel(label) //
-                .withAutoUpdatePolicy(channelType.getAutoUpdatePolicy());
+                .withAutoUpdatePolicy(autoUpdatePolicy);
```

Comment:
Can you add a test-case for this, too? The test should show that

- no policy is set if neither the channel-definition nor the channel-type-definition set a policy
- if either channel- or channel-type-definition contain a policy, it is used
- if both contain a policy, the channel-definition takes precedence

Reply:
Just to be sure, you mean a new test class, right? I didn't find any existing coverage.

Reply:
I'm afraid I need a bit of guidance here. I found some integration tests that seem related:
https://github.com/openhab/openhab-core/blob/3624682c1e2574d395b9fd0e81592404365722b8/itests/org.openhab.core.thing.tests/src/main/java/org/openhab/core/thing/internal/ThingManagerOSGiJavaTest.java#L347-L382

Did you mean to add new tests here? In that case, how can I build and run those tests?